src/test/clojure/clara/test_accumulators.clj

```diff
@@ -59,10 +59,15 @@
         session (-> empty-session
                     (insert (->Temperature 30 "MCI"))
                     (insert (->Temperature 10 "MCI"))
-                    (insert (->Temperature 80 "MCI")))
+                    (insert (->Temperature 80 "MCI"))
+                    fire-rules)
```

Comment:
Short version: I do have concerns, but the current behavior isn't reliable anyway as discussed in https://github.com/cerner/clara-rules/issues/268#issue-208938988 specifically the first bullet, isn't a public contract, and simply changing it seems like the lesser of evils to me.  Making using this sort of caching or non-caching a user-facing option would be one possibility, but I hesitate to permanently add the complication of another user-facing option solely for passivity when the original behavior was unreliable anyway, wasn't a part of the API contract, and the uplift for anyone who runs into the issue should be pretty trivial.  That said, it is possible that others think differently/see other concerns  which is a major reason why I solicited input on the parent issue.